### PR TITLE
[ New App ] - Konfidency

### PR DESCRIPTION
**Pull request by gabrielMatosBoubee**
## What is this contribution about?

This is about a new app. Konfidency

https://www.embelleze.com/shampoo-novex-hidratante-regenerador-recarga-de-queratina-brasileira-07566/p?skuId=2145371093

### CODE REVIEW

**Review by mcandeia — APPROVED** [review]

File: konfidency/loaders/productDetailsPage.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      return {
+        ...productDetailsPage,
+        product: {
+          ...productDetailsPage.product,
+          aggregateRating,
+          review,
+        },
+      };
+    } catch (error) {
+      console.log(error);
```
Comment (mcandeia) [inline konfidency/loaders/productDetailsPage.ts:50]: How about use our logger instead?

use 

import { logger } from "deco/mod.ts"; instead

Comment (mcandeia) [reply konfidency/loaders/productDetailsPage.ts:50]: Remember, you need to log a STRING, so you should extract the error message.

File: konfidency/loaders/productDetailsPage.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      return { ...productDetailsPage };
```
Comment (mcandeia) [inline konfidency/loaders/productDetailsPage.ts:51]: why did you need to shallow copy here?

File: konfidency/loaders/productDetailsPage.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  ctx: AppContext,
+): ExtensionOf<ProductDetailsPage | null> {
+  const { api, customer } = ctx;
+  return async (productDetailsPage: ProductDetailsPage | null) => {
+    if (!productDetailsPage) {
+      return null;
+    }
+
+    if (!productDetailsPage.product.inProductGroupWithID) {
+      return { ...productDetailsPage };
```
Comment (mcandeia) [inline konfidency/loaders/productDetailsPage.ts:29]: why did you need to shallow copy here?

File: konfidency/utils/client.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import type { PDPReview } from "./types.ts"
+
```
Comment (mcandeia) [inline konfidency/utils/client.ts:2]:
```suggestion
```

**Review by mcandeia — APPROVED** [review]
Fix, then shipit

File: konfidency/loaders/productDetailsPage.ts (hunk 2 above)
Comment (gabrielMatosBoubee) [reply konfidency/loaders/productDetailsPage.ts:51]: To return the productDetails without review. I think is better render the PDP component with or without review.
